### PR TITLE
final fixes

### PR DESCRIPTION
**Pull request by shrutigxuriti**

### CODE REVIEW

**Review by kunal-xu — APPROVED** [review]

**Comment by kunal-xu** [comment]
@shrutigxuriti please push updates from pre login kyc pages.

File: src/modules/auth/pages/ReUpdateKycPreview.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ReUpdateKycPreview: React.FC<ReUpdateKycPreviewProps> = ({ navigation }) =
     const [loader, showLoader] = useState(true);
 
     useEffect(() => {
-        AsyncStorage.getItem('USER').then(result => {
+        AsyncStorage.getItem('VGUSER').then(result => {
```
Comment (kunal-xu) [inline src/modules/auth/pages/ReUpdateKycPreview.tsx:26]: What are storing in VGUSER?

Comment (shrutigxuriti) [reply src/modules/auth/pages/ReUpdateKycPreview.tsx:26]: user details

Comment (kunal-xu) [reply src/modules/auth/pages/ReUpdateKycPreview.tsx:26]: ok. can you check whether we can avoid this? Can we store it with USER key and use it afterward or do we really have to create a new key VGUSER?

Comment (shrutigxuriti) [reply src/modules/auth/pages/ReUpdateKycPreview.tsx:26]: we can store it in USER key but anyways after the completion or beginning of KYC i am emptying the VGUSER

Comment (kunal-xu) [reply src/modules/auth/pages/ReUpdateKycPreview.tsx:26]: great

**Pull request merged**